### ISSUE TITLE
list_clients() reports the wrong dBm level for clients

### ISSUE THREAD

**Issue by scottchiefbaker** [Art-of-WiFi/UniFi-API-client#58]
Calling `list_clients()` returns information about: signal, rssi, noise, etc... I'm trying to coordinate the dBm levels I see in the Unifi controller UI with what shows up via the API.

I have a client in the Unifi Controller which shows: "99% (-43 dBm)" but what comes back via `list_clients()` and the API is:
| Key | Value|
|---|---|
|Signal| -58|
|RSSI| 51|
|Noise| -109|

Is there a calculation I need to run to get the actual dBm?

**Comment by malle-pietje** [comment]
I’m afraid you’ll have to ask Ubiquiti for clarification on this. The question isn’t actually related to the API client itself.

**Comment by malle-pietje** [comment]
This link may provide some insights:
http://madwifi-project.org/attachment/wiki/UserDocs/RSSI/Converting_Signal_Strength.pdf?format=raw

**Comment by scottchiefbaker** [comment]
Aha... That is very helpful. I'll use the RSSI - 95 as the basis for my dBm calculations for now because it's simple. If anyone finds the exact formula Unifi uses I'd be curious to see that too.

**Comment by scottchiefbaker** [comment]
I just did some real world comparison between the controller UI and this calculation. It appears from my spot checking that Unifi does `RSSI - 95` for the controller UI signal level. 

The column in the WebUI is "signal" and there is a field in the API call "signal" but they are totally different (close-ish but different). I have added code to my API call

    $ret['x_signal'] = $ret['rssi'] - 95;

And things line up exactly. It may be worth adding to your code as well. I never would have figured this out without that PDF. Thank you!
